gravitee-apim-console-webui/src/management/api/general-info/api-general-info-export-v4-dialog/api-general-info-export-v4-dialog.component.html

```diff
@@ -42,11 +42,7 @@ <h5>Include additional data</h5>
         <div>
           Export your API Definition as a Kubernetes Custom Resource Definition and start using our Kubernetes Operator to manage your API
           declaratively.
-          <a
-            href="https://documentation.gravitee.io/apim/getting-started/install-guides/install-on-kubernetes"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
+          <a href="https://documentation.gravitee.io/apim/guides/gravitee-kubernetes-operator" target="_blank" rel="noopener noreferrer">
```

Comment:
I'm not sure if direct URL it's good for white brand like seligo 🤷‍♂️

Reply:
👍 Good catch, any thought @jmcx ?

Reply:
@ThibaudAV on the assumption that a bad link is better than a dead link, i'll merge and bring this offline to @jmcx